### ISSUE TITLE
Gerunds in Spanish don't appear to be conjugating correctly

### ISSUE THREAD

**Issue by pbaldridgework** [SekouD/mlconjug#82]
`import mlconjug`
`default_conjugator = mlconjug.Conjugator("es")`
`conjug_info = default_conjugator.conjugate("correr").conjug_info`
`gerund = conjug_info["Gerundio"]["Gerundio"]['']`
returns "corrido" not "corriendo". The same is true for other verbs (hablar, andar, etc.)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Latest version of mlconjug

**Comment by SekouDiaoNlp** [comment]
Hi @pbaldridgework , I am @SekouDiaoNlp, formerly on GitHub as @SekouD and I am the original author of mlconjug. I created a new GitHub account at @SekouDiaoNlp as my former account was mostly for my academic projects, while my new account @SekouDiaoNlp is more focused on my professional projects as well as my open-source projects.
I have just released a new version of mlconjug called mlconjug3, to indicate that I dropped support for Python 2.x as it has been deprecated by the end of 2019.

To install it just run :
pip install mlconjug3


And to answer your initial bug request, the bug has been fixed in version 3.7.1 of mlconjug3

Have a nice day and thank you for using mlconjug3.

Cheers!
